test/scaling.jl

```diff
@@ -464,112 +466,122 @@
         @testset "NamedTuple" begin
             nt = (a = [0.0, -0.5, 0.5], b = [1.0, 0.0, 2.0])
 
-            arr_expected = [[-0.55902, -1.11803, 0.0], [0.55902, -0.55902, 1.67705]]
-            nt_expected = (a = [0.0, -1.0, 1.0], b = [0.0, -1.0, 1.0])
-
             @testset "Non-mutating" begin
+                expected = [[-0.55902, -1.11803, 0.0], [0.55902, -0.55902, 1.67705]]
+
                 scaling = MeanStdScaling(nt)
-                @test FeatureTransforms.apply(nt, scaling) ≈ arr_expected atol=1e-5
-                @test scaling(nt) ≈ arr_expected atol=1e-5
+                result = FeatureTransforms.apply(nt, scaling)
+                @test result isa NamedTuple{(:Column1, :Column2)}
+                @test collect(result) ≈ expected atol=1e-5
+
+                result = scaling(nt)
+                @test result isa NamedTuple{(:Column1, :Column2)}
+                @test collect(result) ≈ expected atol=1e-5
             end
 
             @testset "Mutating" begin
+                expected = [[-0.55902, -1.11803, 0.0], [0.55902, -0.55902, 1.67705]]
                 scaling = MeanStdScaling(nt)
                 _nt = deepcopy(nt)
                 FeatureTransforms.apply!(_nt, scaling)
                 @test _nt isa NamedTuple{(:a, :b)}
-                @test collect(_nt) ≈ collect(arr_expected) atol=1e-5
+                @test collect(_nt) ≈ expected atol=1e-5
             end
 
-            @testset "cols = $c" for c in (:a, :b)
-                scaling = MeanStdScaling(nt; cols=[c])
-                nt_mutated = NamedTuple{(Symbol("$c"), )}((nt_expected[c], ))
-                nt_expected_ = merge(nt, nt_mutated)
+            @testset "cols = :a" begin
+                scaling = MeanStdScaling(nt; cols=:a)
 
-                @test FeatureTransforms.apply(nt, scaling; cols=[c]) ≈ [collect(nt_expected_[c])] atol=1e-14
-                @test FeatureTransforms.apply(nt, scaling; cols=c) ≈ collect(nt_expected_[c]) atol=1e-14
-                @test scaling(nt; cols=[c]) ≈ [collect(nt_expected_[c])] atol=1e-14
+                expected = (Column1 = [0.0, -1.0, 1.0], )
+                @test FeatureTransforms.apply(nt, scaling; cols=:a) == expected
+                @test FeatureTransforms.apply(nt, scaling; cols=[:a]) == expected
+                @test scaling(nt; cols=:a) == expected
 
                 _nt = deepcopy(nt)
-                FeatureTransforms.apply!(_nt, scaling; cols=[c])
+                FeatureTransforms.apply!(_nt, scaling; cols=:a)
                 @test _nt isa NamedTuple{(:a, :b)}  # before applying `collect`
```

Comment:
```suggestion
```
I think it's unnecessary due to change below

Reply:
yep, a bunch of tests are now unnecessary